### DISCUSSION TITLE
Purpose

### DISCUSSION THREAD

**Discussion by maurelio-cky** [balzack/databag#96] [General]
hey,

what's the purpose of the 01 button, and can I turn it off? Also, just out of curiosity, would it be possible for the admin to create channels visible to everyone? maybe an additional tab for public channels or something or did I simply miss it?

kind regards 

![Screenshot_20240501_194903](https://github.com/balzack/databag/assets/120311557/92adf70a-bde6-42fc-b63b-b3aab7de0161)

**Comment by balzack** [comment]
The '01' button allows the user to attach a binary file. It's a catch all for anything like PDF or XLS.

Currently there is no option to suppress the button, but it should be easy to add if you would like.

**Reply by maurelio-cky** [reply]
If it doesn't bother you too much. would be great of course.

**Comment by balzack** [comment]
I will include the changes in the next release, but it won't be immediate. There will be an upcoming mobile release, but it will also require the server release which will follow.

I am always happy to hear about people using the project and ways that I can make it better. Thank you.

**Comment by maurelio-cky** [comment]
don't mind me. Just take your time.
